sdk/python/feast/big_query_source.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 The Feast Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Optional
+
+
+class BigQuerySource:
+    """
+    Represents a BigQuery table reference or BigQuery query that returns a set of features.
+    """
+
+    def __init__(
+        self,
+        table_ref: Optional[str],
+        event_timestamp_column: str,
+        created_timestamp_column: Optional[str],
+        field_mapping: Optional[Dict[str, str]],
```

Comment:
I think what isn't clear here is what direction of the field mapping.

Reply:
oh yeah I've been assuming it's key=table, value=feature

Reply:
Ah, I was thinking the reverse.

Reply:
Hm, intuitively table->feature makes more sense to me but I may be in the minority with that viewpoint - do we do the reverse in Feast today?

Reply:
I dont think it matters, as long as we document it.